docker/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+set -eu
+
+# Compiler packages to install
+
+case "$DMD" in
+    1.*    ) PKG="dmd1=$DMD-$DIST" ;;
+    2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;;
+    2.*    ) if [ $(echo $DMD | cut -d. -f2) -ge 077 ]; then
+                PKG="dmd-compiler=$DMD dmd-tools=$DMD libphobos2-dev=$DMD"
+             else
+                PKG="dmd-bin=$DMD libphobos2-dev=$DMD"
+             fi ;;
+    *      ) echo "Unknown \$DMD ($DMD)" >&2; exit 1 ;;
+esac
+
+# Compiler flags to supply to makd
+
+case "$DMD" in
+    1.*    ) DVER=1; DC=dmd1 ;;
+    2.*.s* ) DVER=2; DC=dmd-transitional ;;
+    2.*    ) DVER=2; DC=dmd ;;
+esac
```

Comment:
This can be reused from beaver: https://github.com/sociomantic-tsunami/beaver/blob/v0.x.x/lib/dlang.sh

You can include the lib `. beaver/lib/dlang.sh` and then call the provided functions.

Reply:
Will do :+1:

Reply:
Updated